corehq/apps/reports/standard/cases/basic.py

```diff
@@ -321,10 +322,6 @@ def rows(self):
 
     def date_to_json(self, date):
         if date:
-            return date.strftime('%Y-%m-%d %H:%M:%S')
-            # temporary band aid solution for http://manage.dimagi.com/default.asp?80262
-            # return tz_utils.adjust_datetime_to_timezone(
-            #     date, pytz.utc.zone, self.timezone.zone
-            # ).strftime('%Y-%m-%d %H:%M:%S')
+            return PhoneTime(date, self.timezone).user_time(self.timezone).done().strftime('%Y-%m-%d %H:%M:%S')
```

Comment:
Yeah the other benefit is that this code continues to work after flipping `TIMEZONE_DATA_MIGRATION_COMPLETE`, actually doing the timezone conversion once that's necessary.